pkg/consensus/client.go

```diff
@@ -14,6 +14,9 @@ import (
 	"github.com/protolambda/eth2api/client/validatorapi"
 	"github.com/protolambda/zrnt/eth2/beacon/bellatrix"
 	"github.com/protolambda/zrnt/eth2/beacon/common"
+	consensustypes "github.com/prysmaticlabs/prysm/v3/consensus-types/primitives"
```

Comment:
we could have just alias'd this to `types` and reduced the code churn by quite a bit